tests/integration/tier0/non-destructive/rollback-handling/test_rollback_handling.py

```diff
@@ -94,6 +94,24 @@ def terminate_and_assert_good_rollback(c2r):
     c2r.expect("Rollback: Remove installed certificate", timeout=120)
 
 
+@pytest.mark.test_yum_plugin_local
+def test_proper_rhsm_clean_up(shell, convert2rhel):
```

Comment:
```suggestion
@pytest.mark.test_polluted_yumdownloader_output_by_yum_plugin_local
def test_polluted_yumdownloader_output_by_yum_plugin_local(shell, convert2rhel):
```
Copy&paste error in function name, make the test name more explanatory